internal/pkg/manifest/workload_ecs.go

```diff
@@ -91,15 +91,37 @@ type AlarmArgs struct {
 	MemoryUtilization *float64 `yaml:"memory_utilization"`
 }
 
-// DeploymentConfiguration represents the deployment strategies for a service.
+// WorkerAlarmArgs represents specs of CloudWatch alarms for Worker Service deployment rollbacks.
+type WorkerAlarmArgs struct {
+	AlarmArgs       `yaml:",inline"`
+	MessagesDelayed *int `yaml:"messages_delayed"`
+}
+
+// DeploymentControllerConfig represents deployment strategies for a service.
+type DeploymentControllerConfig struct {
+	Rolling *string `yaml:"rolling"`
+}
+
+// DeploymentConfiguration represents the deployment config for an ECS service.
 type DeploymentConfiguration struct {
-	Rolling        *string                    `yaml:"rolling"`
-	RollbackAlarms Union[[]string, AlarmArgs] // `yaml:"rollback_alarms"` 
+	DeploymentControllerConfig `yaml:",inline"`
+	RollbackAlarms             Union[[]string, AlarmArgs] // `yaml:"rollback_alarms"`
+	// The rollback_alarms manifest field is a no-op until the EDS-CFN ABR bug is fixed.
```

Comment:
What are we doing to prevent user from using this field?

Reply:
commented out the yaml, so if they do use the field, nothing happens. users wouldn't know to try using it!